src/shoppingCart/Demo.java

```diff
@@ -0,0 +1,98 @@
+package shoppingCart;
+
+public class Demo {
+
+	public static void main(String[] args) {
+		// Some examples
+
+		// Products
+		Store wallMart = new Store();

```

Comment:
Let's delegate all this product creation to a Builder or Factory. You could also model a Data Access Objet to encapsulte all this logic.